apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -603,11 +605,82 @@ public Response getUserFilledOrders(@Parameter(description = "Number of first N
+                                                     @Parameter(description = "Order id (signed/unsigned int64/long) ") @QueryParam("orderId") String order) {
         long accountId = Convert.parseAccountId(account);
         long orderId = Convert.parseLong(order);
         List<ExchangeContract> contracts = service.getVersionedContractsByAccountOrder(accountId, orderId);
         return Response.ok(contractConverter.convert(contracts)).build();
     }
 
+    @GET
+    @Path("/eth/addresses")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(tags = {"dex"}, description = "Get all user addresses on the smart contract. ",
+        responses = @ApiResponse(description = "List of user addresses", responseCode = "200",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))))
+    public Response getAllUsers() {
+        List<String> addresses;
+        try {
+            addresses = service.getAllUsers();
+        } catch (AplException.ExecutiveProcessException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+        return Response.ok(addresses).build();
+    }
+
+    @GET
+    @Path("/eth/filledOrders")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(tags = {"dex"}, description = "Get all users filled orders on the smart contract. ",
+        responses = @ApiResponse(description = "List of user filled orders", responseCode = "200",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))))
+    public Response getAllUsersFilledOrders() {
+
+        List<AddressEthDepositsInfo> addressEthDepositsInfos;
+        try {
+            addressEthDepositsInfos = service.getAllFilledOrders();
+        } catch (AplException.ExecutiveProcessException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+
+        return Response.ok(addressEthDepositsInfos).build();
+    }
+
+    @GET
+    @Path("/eth/expiredSwaps")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(tags = {"dex"}, description = "Get all users expired swaps on the smart contract. ",
+        responses = @ApiResponse(description = "List of user expired swaps", responseCode = "200",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))))
+    public Response getAllUsersExpiredSwaps() {
+
+        List<AddressEthExpiredSwaps> addressEthExpiredSwaps;
+        try {
+            addressEthExpiredSwaps = service.getAllExpiredSwaps();
+        } catch (AplException.ExecutiveProcessException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+
+        return Response.ok(addressEthExpiredSwaps).build();
+    }
+
+
+    @GET
+    @Path("/eth/activeDeposits")
```

Comment:
use active-deposits instead of activeDeposits since it belongs to REST best practices